financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/pom.xml

```diff
@@ -203,6 +203,7 @@
                             com.fasterxml.jackson.core;version="${jackson.databinding.version}",
                             org.owasp.encoder;version="${encoder.wso2.version}"
                         </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
Why is this needed?

Reply:
As discussed offline had to add this to solve dependency not identifying issue.